components/Collections/examples/CollectionsSimpleSwiftDemo.swift

```diff
@@ -26,6 +26,10 @@ class CollectionsSimpleSwiftDemo: MDCCollectionViewController {
   override func viewDidLoad() {
     super.viewDidLoad()
 
+    if #available(iOS 11.0, *) {
```

Comment:
I don't know if this will compile on older XCode versions.

Reply:
I know it will on XCode 8.3.3.

Reply:
This was added in swift 2, so it should be OK in Xcode 8. It is worthwhile to test on Xcode 7, but I [believe it is supported](https://developer.apple.com/library/content/documentation/Xcode/Conceptual/WhatsNewXcode-Archive/Articles/xcode_7_0.html).

Reply:
Ok, I am letting it for now.